CppCoreGuidelines.md

```diff
@@ -14456,7 +14456,7 @@ It simply has nothing to do with concurrency.
         if (int n = free_slots--) return &pool[n];
     }
 
-Here we have a problem:
+Here, we have a problem:
```

Comment:
I don't think commas after these "here"s are warranted. These aren't discourse markers; the text isn't saying "Here, would you look at that? We have a problem!" It's saying "Here, on that line above, we have a problem."

Though I suppose it's not entirely wrong either way https://english.stackexchange.com/questions/321371